base/applications/cmdutils/find/lang/ru-RU.rc

```diff
@@ -8,11 +8,11 @@ FIND [/V] [/C] [/N] [/I] [/OFF[LINE]] ""строка"" [[диск:][путь]и
   /C         Вывод только общего числа строк, содержащих заданную строку.\n\
   /N         Вывод номеров отображаемых строк.\n\
   /I         Поиск без учета регистра символов.\n\
-  /OFF[LINE] Do not skip files that have the offline attribute set.\n\
+  /OFF[LINE] Не пропускать файлы, у которых установлен атрибут offline.\n\
   ""строка""   Искомая строка.\n\
   [диск:][путь]имя_файла\n\
              Один или несколько файлов, в которых выполняется поиск."
     IDS_NO_SUCH_FILE "FIND: %s: Файл не существует.\n"
     IDS_CANNOT_OPEN "FIND: %s: Невозможно открыть файл.\n"
-    IDS_INVALID_SWITCH "FIND: Invalid switch\n"
+    IDS_INVALID_SWITCH "FIND: Неправильный ключ\n"
```

Comment:
In this context, "switch" or "key" means a keyword for searching the file. So it's looks better as follows:
```suggestion
    IDS_INVALID_SWITCH "FIND: Неправильный поисковый запрос\n"
```
The "key" might also mean "Activation Key", "Product Key", so it might cause some misunderstanding. :slightly_smiling_face: